package-lock.json

```diff
@@ -3562,8 +3552,7 @@
     "strip-json-comments": {
       "version": "2.0.1",
       "resolved": "https://registry.npmjs.org/strip-json-comments/-/strip-json-comments-2.0.1.tgz",
-      "integrity": "sha1-PFMZQukIwml8DsNEhYwobHygpgo=",
-      "dev": true
```

Comment:
some of these are missing optional. is this intentional?

Reply:
Just means that they are used by some other dependency which is not optional.